ci/download_latest_bundle.py

```diff
@@ -14,23 +14,14 @@
 import os
 
 from monai.bundle import download
-from utils import get_checksum, get_hash_func, get_latest_version, get_version_checksum
+from utils import get_latest_version
```

Comment:
For huggingface hosting, each bundle's each version has it's commit, no need to use checksum to ensure consistency. cc @ericspod @KumoLiu